tests/playbooks/roles/cf_deploy/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: Set String for CloudFormation Stack ID
   set_fact:
-    cloudformation_stack_id: "ansible-cloudformation-{{ lookup('pipe','date +%Y-%m-%d-%H-%M-%S') }}"
+    cloudformation_stack_id: "ACF-{{ lookup('pipe','date +%Y-%m-%d-%H-%M-%S') }}"
```

Comment:
did you shorten the name because there is an issue with a longer stack name?

Reply:
Yes

Reply:
Also I thought to put more informative names to stacks instead of "ansible-cloudformation-*"

Reply:
I Agree. but ACF is not informative at all :)
BTW, did you informed Shira about this issue? she should know that

Reply:
I Agree. but ACF is not informative at all :)
BTW, did you informed Shira about this issue? she should know that